packages/plugin-types/API.md

```diff
@@ -135,6 +147,21 @@ export type MainStackNameResolver = {
     resolveMainStackName: () => Promise<string>;
 };
 
+// @public (undocumented)
+export type PackageManagerController = {
+    getWelcomeMessage: () => string;
+    initializeProject: () => Promise<void>;
+    initializeTsConfig: (targetDir: string) => Promise<void>;
+    installDependencies: (packageNames: string[], type: DependencyType) => Promise<void>;

```

Comment:
```suggestion
    installDependencies: (packageNames: string[], type: 'dev' | 'prod') => Promise<void>;
```

I'd inline this parameter type for now. Less is better.

Reply:
Done